test/distributed/fsdp/test_fsdp_hybrid_shard.py

```diff
@@ -54,6 +52,7 @@ def patch_allreduce(new_allreduce):
     finally:
         dist.all_reduce = orig_ar
 
+
```

Comment:
Admittedly nitpicky, but these sort of empty line changes also make the logical changes to the PR to review in my experience.